sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -98,36 +104,298 @@ const client = new TextAnalyticsClient("<endpoint>", new DefaultAzureCredential(
+
+### Recognize Linked Entities
+
+A "Linked" entity is one that exists in a knowledge base (such as Wikipedia). The `recognizeLinkedEntities` operation can disambiguate entities by determining which entry in a knowledge base they likely refer to (for example, in a piece of text, does the word "Mars" refer to the planet, or to the Roman god of war). Linked entities contain associated URLs to the knowledge base that provides the definition of the entity.
+
+```javascript
+const { TextAnalyticsClient, TextAnalyticsApiKeyCredential } = require("@azure/ai-text-analytics");
+
+const client = new TextAnalyticsClient(
+  "<endpoint>",
+  new TextAnalyticsApiKeyCredential("<API key>")
+);
+
+const documents = [
+  "Microsoft was founded by Bill Gates and Paul Allen.",
+  "Easter Island, a Chilean territory, is a remote volcanic island in Polynesia.",
+  "I use Azure Functions to develop my product."
+];
+
+async function main() {
+  const results = await client.recognizeLinkedEntities(documents, "en");
+
+  for (const result of results) {
+    if (result.error === undefined) {
+      console.log(" -- Recognized linked entities for input", result.id, "--");
+      for (const entity of result.entities) {
+        console.log(entity.name, "(URL:", entity.url, ", Source:", entity.dataSource, ")");
+        for (const match of entity.matches) {
+          console.log("  Occurrence:", "\"" + match.text + "\"", "(Score:", match.score, ")");
+        }
+      }
+    } else {
+      console.error("Encountered an error:", result.error);
+    }
+  }
+}
+
+main();
+```
+
+### Extract Key Phrases
+
+Key Phrase extraction identifies the main talking points in a document. For example, given input text "The food was delicious and there were wonderful staff", the service returns "food" and "wonderful staff".
+
+```javascript
+const { TextAnalyticsClient, TextAnalyticsApiKeyCredential } = require("@azure/ai-text-analytics");
+
+const client = new TextAnalyticsClient(
+  "<endpoint>",
+  new TextAnalyticsApiKeyCredential("<API key>")
+);
+
+const documents = [
+  "Redmond is a city in King County, Washington, United States, located 15 miles east of Seattle.",
+  "I need to take my cat to the veterinarian.",
+  "I will travel to South America in the summer."
+];
+
+async function main() {
+  const results = await client.extractKeyPhrases(documents, "en");
+
+  for (const result of results) {
+    if (result.error === undefined) {
+      console.log(" -- Extracted key phrases for input", result.id, "--");
+      console.log(result.keyPhrases)
+    } else {
+      console.error("Encountered an error:", result.error);
+    }
+  }
+}
+
+main();
+```
+
+### Detect Language
+
+Determine the language of a piece of text.
+
+The `countryHint` parameter is optional, but can assist the service in providing correct output if the country of origin is known. If it is not provided, or the value `"none"` is given, then a default model that does not assume a country of origin will be used.
```

Comment:
Service will use "US" if no countryHint is provided

Reply:
I think there is actually a difference. My understanding is that some weight is given to languages that are common in the USA but that it's different from actually passing `"us"` to the service.

Reply:
There may be more happening behind the scenes then I understand, but I think we should align with what the service is saying in documentation: https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/quickstarts/text-analytics-sdk?tabs=version-3&pivots=programming-language-csharp

Reply:
That was supposed to link to the Tip in the language detection section

Reply:
oh interesting, so it's saying that if no hint is provided at all, then it uses "us", but that explicitly sending "" or "none" in our case will use the no-country model.